docs/extend/plugins.md

```diff
@@ -70,6 +70,8 @@ The following plugins exist:
 source volume plugin that provides multi-tenant, persistent, distributed storage
 with intent based consumption using ceph underneath.
 
+* The [Quobyte Volume Plugin](https://github.com/quobyte/docker-volume) connects Docker to [Quobyte](http://www.quobyte.com/containers)'s data center file system, a general-purpose scalable and fault-tolerant storage platform.
```

Comment:
Can you move this above the "REX-Ray" plugin? I realize the list is already out of order, but we should try to keep it sorted alphabetically

After moving, please commit with `git commit --amend`, and do a *force* push (`git push -f ...`), that should update the PR on github